sig-operations/README.md

```diff
@@ -26,16 +26,25 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * Lars Kellogg-Stedman (**[@larsks](https://github.com/larsks)**), Red Hat
 
 ## Contact
-- Slack: [#operations](https://operatefirst.slack.com/messages/operations)
+- Slack: [#sig-operations](https://operatefirst.slack.com/messages/sig-operations)
 - [Mailing list](community@lists.operate-first.cloud)
 - [Open Community Issues/PRs](https://github.com/open-services-group/community/labels/sig%2Foperations)
```

Comment:
Do we want to leave the community mailing list and issues repository here in the sig-operations document? I worry that people will end up creating operational issues in the community issues repository.

Reply:
hrmm yeah this part is pulled from the templates here: https://github.com/operate-first/community/tree/main/generator 

I think we need a separate issue to revisit the generator files there. I initially pulled them from the open-services-group (which in turn probably got them from the kubernetes org). Either we replace references to osg with op1st or remove them entirely.

Reply:
I think for now we should either remove the mailing list and issues link, or replace them with ones appropriate to sig-operations.